README.md

```diff
@@ -53,7 +54,7 @@ Ensure the IAM user or role for these credentials has permission to create these
 This will be addressed in the future.
 
 
-### Install
+### Installation
 
 Check out Romana repository.
 ```bash
```

Comment:
'sh-session' for copy-paste from a terminal session (including prompt and command output)
'bash' for plain commands that a user should execute.